tests/benchmark/snappy/dynamic/PerfRun.conf.template1

```diff
@@ -0,0 +1,109 @@
+#Do not change the way of queries, sparkProperties, sparkSqlProperties. Just change the values inside strings
+
+#queries to be run
+queries="\"1-2-3-4-5-6-7-8-9-10-11-12-13-14-15-16-17-18-19-20-22-21\""
+#queries="\"2-5-7-10-11-14-22\""
+#queries=10
+#queries="\"1-3-14\""
+
+
+# Common properties
+#commonProperties="-XX:NewRatio=3 -spark.local.dir=/vol1/users/perfuser/tmp"
+#commonProperties="-XX:NewRatio=3 -snappydata.sql.hashJoinSize=2g -snappydata.enable-experimental-features=true -spark.local.dir=/vol1/users/perfuser/tmp"
+#commonProperties="-XX:NewRatio=3 -spark.sql.autoBroadcastJoinThreshold=50000000 -spark.local.dir=/vol1/users/perfuser/tmp"
+commonProperties="-XX:NewRatio=3 -snappydata.sql.hashJoinSize=2g -spark.sql.autoBroadcastJoinThreshold=50000000 -snappydata.enable-experimental-features=true -spark.local.dir=/vol1/users/perfuser/tmp"
+#spark Properties spefied in lead
+#spark-executor-cores has nothing to do with CPU cores available.
+#sparkProperties="-J-Xmx2g -spark.network.timeout=300s -spark.driver.maxResultSize=2g -spark.shuffle.sort.bypassMergeThreshold=28"
+sparkProperties="-heap-size=10g -memory-size=40g $commonProperties"
+
+#Spark Sql properties are specified while executing query
+#spark.sql.shuffle.partitions=${shufflePartitions},spark.sql.inMemoryColumnarStorage.compressed=${inMemoryColumnarStorageCompressed}
+#spark.sql.inMemoryColumnarStorage.compressed=false
+sparkSqlProperties="\"\""
+
+#location of checkout
+#SnappyData=/data1/users/perfuser/snappyBuilds/1.0.0/snappydata-1.0.0-bin
```

Comment:
Can we replace /data1/users/perfuser/ with an environment variable $DATA_DIR ?

```
if [ -z "$DATA_DIR" ] 
then
	echo "Please set \$DATA_DIR"
else
	DATA_DIR=~; echo "Setting up base directory at $DATA_DIR"
```

Reply:
I think, setting up the path variables in conf would be better here instead of environment variable. For each PerfRun, the conf template has to be modified into PerfRun.conf file according to the test run requirements.
I will just remove the hard coded paths and specify the description for same in template. Users will have to create their own copy of PerfRun.conf for running the test.